helm/terraform-job/templates/terraform-role.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.serviceAccount.roleName }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
`helm.sh/hook-weight: "-10"`

Reply:
Made sure all the templates have the hook weight ordering with 49bfe48.